homeassistant/util/aiohttp.py

```diff
@@ -48,6 +48,6 @@ def serialize_response(response: web.Response) -> Dict[str, Any]:
     """Serialize an aiohttp response to a dictionary."""
     return {
         'status': response.status,
-        'body': response.body,
```

Comment:
Since this is a generic helper, we should not change its behavior to make it more shaped for one implementation. If we want that, we should move this function inside the cloud integration.